test/integration/steps/test_parent_steps.py

```diff
@@ -41,6 +42,22 @@ def a_change_event_is_valid():
     return context
 
 
+@given(parsers.parse('a Changed Event with changed "{contact}" is valid'), target_fixture="context")
+def a_changed_contact_event_is_valid(contact):
+    context = {}
+    context["change_event"] = create_change_event()
+    if contact == "website":
+        context["change_event"]["Contacts"][0]["ContactValue"] = "www.my-test-website.com"
```

Comment:
The problem with this at the moment is that test will only pass the first time as a change will not be generated on subsequent runs as these values will already be committed. Instead you could have two options for each, flip between them based on current state

```
if  context["change_event"]["Contacts"][0]["ContactValue"] == "www.test1.com":
   context["change_event"]["Contacts"][0]["ContactValue"]  = "www.test2.com"
else:
   context["change_event"]["Contacts"][0]["ContactValue"]  = "www.test1.com"
```

Reply:
may be  you can consider this as well .  you don't need to worry about index .
contact = 'website' # or Telephone
item = list(filter(lambda x:x["ContactMethodType"] == contact, change_event['Contacts']))
value = item[0]["ContactValue"]

Reply:
Resolved.